app/src/redux-resources/analytics/hooks/useCameraAnalytics.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  ANALYTICS_CAMERA_ENABLEMENT_KIND,
+  ANALYTICS_CAMERA_SETTINGS_KIND,
+  ANALYTICS_IMAGE_CAPTURE_KIND,
+  ANALYTICS_LIVE_FEED_KIND,
+  ANALYTICS_PHOTO_ACCESS,
+  useTrackEvent,
+} from '/app/redux/analytics'
+
+import type { RobotType } from '@opentrons/shared-data'
+
+export const SOURCE_RUN_RECORD = 'runRecord'
+export const SOURCE_ROBOT_SETTINGS = 'robotSettings'
```

Comment:
```suggestion
export const SOURCE_ROBOT_SETTINGS = 'robotSettings' as const
```
The `as const` tells Typescript that we aren't passing in a string here (the wide type) but the narrowest type possible: the string that is exactly `robotSettings`.